doc/source/installation.rst

```diff
@@ -142,10 +142,16 @@ source code modified.
  Installing |Percona Server| from a Source Tarball
```

Comment:
This section is about source tarballs, binary tarball discussion should go elsewhere (create a new section if needed)